digicampipe/scripts/concatenate.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+"""
+concatenate the input fits files to one output
+
```

Comment:
Could you maybe elaborate a bit more on the use case? I do not fully understand what concat means. Along which axis are the file concatted? 
I assume there is one typical use-case which can be used as an example?

Reply:
I expanded a bit the description.
The idea is when we have several Tables saved as fit files, we can merge them (get a table with the rows from all files)

Reply:
Ah I see, thanks